### PR TITLE
Added default classifier and mapper to EWS O365

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26808#event-3594329084

## Description
* From 6.0 - the same classifier and mapper will be used for both integration
* Created a 5.9.9 classifier for EWS O365

### CODE REVIEW

File: Packs/EWS/Classifiers/classifier-EWS_v2_CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```
Comment (ronykoz) [inline Packs/EWS/Classifiers/classifier-EWS_v2_CHANGELOG.md:1]: please use the releasenotes, not the changelog

File: Packs/EWS/Classifiers/classifier-mapper-incoming-EWS_v2_CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```
Comment (ronykoz) [inline Packs/EWS/Classifiers/classifier-mapper-incoming-EWS_v2_CHANGELOG.md:1]: same here

**Comment by reutshal** [comment]
> why are we renaming existing ones, please revert

The existing ones will now be used for both integrations, therefore the display name needs to be generic.
The ID cannot be changed because of BC

**Comment by reutshal** [comment]
@roysagi Can you go over the release notes please? Thanks
https://github.com/demisto/content/blob/EWS-mapping/Packs/EWS/ReleaseNotes/1_2_0.md

**Pull request merged**